### PR TITLE
Read.me draft

### PR DESCRIPTION
**Pull request by mareo1997**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+
+**Architecture Diagram**
+![](./imgs/Staging-Module-AD.png)
+
+![](./imgs/Staging-Module-ERD.PNG)
+**ERD Diagram**
+
+Associate status' are emun staging and project. Analysis type are enum strengths, weaknesses, opportunities and threats.
+
+### Technical Requirements
```
Comment (sophiagavrila) [inline README.md]: @mareo1997 Instead of *Techincal Requirements*, write *Technologies Used*

File: README.md [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+- The system will automatically assign associates to a staging manager
+- A manager can create a SWOT analysis for an associate in order to decide actions the associate should prioritize
+
+**Architecture Diagram**
+![](./imgs/Staging-Module-AD.png)
+
+![](./imgs/Staging-Module-ERD.PNG)
+**ERD Diagram**
+
+Associate status' are emun staging and project. Analysis type are enum strengths, weaknesses, opportunities and threats.
```
Comment (sophiagavrila) [inline README.md]: @mareo1997 typo in the word "enum"

File: README.md [hunk 3]
```diff
@@ -1,3 +1,27 @@
...
+- A manager can view all associates assigned to him
+- A manager can view all associates that are new to staging
+- The system will automatically assign associates to a staging manager
+- A manager can create a SWOT analysis for an associate in order to decide actions the associate should prioritize
+
+**Architecture Diagram**
+![](./imgs/Staging-Module-AD.png)
+
+![](./imgs/Staging-Module-ERD.PNG)
+**ERD Diagram**
```
Comment (sophiagavrila) [inline README.md]: @mareo1997 in the directory 'imgs' you can add the png files.  Then the README doc will know what image you're referring to

Comment (sophiagavrila) [reply README.md]: You can add this folder directly into the project -- the same location where the README.md file is.  THEN, put the png images inside this folder,  push your changes, then I will re-review this pull request

Comment (mareo1997) [reply README.md]: I thought I already did that. Like you did in project 1.

Comment (mareo1997) [reply README.md]: For erd diagram if i try to put ./imgs/Staging-Module-ERD.PNG under the label, the format doesn't match. So I just left it there

**Review by sophiagavrila — APPROVED** [review]
LGTM

**Pull request merged**